src/main/java/uk/gov/hmcts/ccd/domain/service/processor/GlobalSearchProcessorService.java

```diff
@@ -176,8 +185,19 @@ private String findValueInMap(String valueToFind, Map<String, JsonNode> mapToSea
                 }
             }
         }
-
         return returnValue;
     }
 
+    private String findDateValueInMap(String valueToFind, Map<String, JsonNode> mapToSearch, String caseTypeID) {
+        String value = findValueInMap(valueToFind, mapToSearch);
+        if (value != null) {
+            try {
+                LocalDate.parse(value, ISO_DATE);
+            } catch (DateTimeParseException e) {
+                log.warn("The value: {} for CaseTypeID: {} is not a Date value in the ISO format", value, caseTypeID);
```

Comment:
We shouldn't be logging the raw value as that is user data: i.e. just the case type will do for an ivestigation to identify the case type.

I think it may be worth mentioning the GlobalSearchProcessorService in some way in the string.

Reply:
done